zprofile

```diff
@@ -0,0 +1,5 @@
+if [ -d "/opt/homebrew" ]; then
+  eval "$(/opt/homebrew/bin/brew shellenv)"
+elif [[ -d /home/linuxbrew ]]; then
```

Comment:
Homebrew on Linux [may also install to `$HOME/.linuxbrew`](https://docs.brew.sh/Homebrew-on-Linux#install):

> The installation script installs Homebrew to /home/linuxbrew/.linuxbrew using sudo if possible and within your home directory at ~/.linuxbrew otherwise.

Can we check for that as well?